### DISCUSSION TITLE
DSD over PCM (DoP) on fifo output

### DISCUSSION THREAD

**Discussion by pepie34** [MusicPlayerDaemon/MPD#1602] [General]
I have reached to setup a working 176400:24:2 PCM fifo output this way:  mpd -> fifo -> snapserver -> LAN -> snapclient for use with topping D50s DAC on a distant raspberry pi.
It works perfectly with PCM but DSD over PCM (DoP) is not working.

I have tried the following options
```
allowed_formats  "176400:24:2 dsd64:=dop"
dop "yes"
```
but this options seems to be only working for alsa output plugin.

Can you consider adding DSD/DoP option for fifo plugin ?
176400:24:2 is the correct transport stream for DSD64

**Comment by MaxKellermann** [comment]
The options you named are not available with the `fifo` output plugin, as you can see in the MPD documentation.
In general, the `fifo` output plugin is considered experimental and not supported. Use at your own risk. Since MPD has a Snapcast output plugin, this kludge is no longer necessary.

**Comment by pepie34** [comment]
I do need a different snapcast server that is not the mpd output plugin internal snapcast server. For example, for using airplay or spotify sources on that snapcast server.
What technically prevent to have these dop options on fifo output plugin ? (beside developing time and willingness)
